Sources/TuistKit/Cache/CacheController.swift

```diff
@@ -114,8 +115,16 @@ final class CacheController: CacheControlling {
         logger.notice("Hashing cacheable targets")
         let hashesByCacheableTarget = try graphContentHasher.contentHashes(for: graph, cacheOutputType: artifactBuilder.cacheOutputType)
 
+        let filteredTargets: [TargetNode] = {
```

Comment:
this is an interesting way to write this but I'd find this more easy to grasp:
```swift
let filteredTargets: [TargetNode]
if let targetsToFilter = targetsToFilter {
   filteredTargets = ...
} else {
  filteredTargets = ...
}
```

but feel free to take leave as-is, this is definitely a bit subjective 🙂

Reply:
If this is not common in the project I prefer to change to your suggested one :)